standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -3179,8 +3179,7 @@ private void deleteDataExcludeCmroot(Path path, boolean ifPurge, boolean shouldE
         wh.deleteDir(path, true, ifPurge, shouldEnableCm);
       }
     } catch (Exception e) {
-      LOG.error("Failed to delete directory: " + path +
-          " " + e.getMessage());
+      LOG.error("Failed to delete directory: " + path, e);
```

Comment:
nit: Its better to use `{}` instead of concatenation in logs

Reply:
Addressed.